packages/snjs/lib/utils.ts

```diff
@@ -75,7 +74,7 @@ export function concatArrays(...args: any[]) {
 /**
  * @returns Whether the value is a function or object
  */
-export function isObject(value: any) {
+export function isObject<T>(value: T|null): boolean {
```

Comment:
Incorrect spacing in T|null

Reply:
Thanks! I had to turn it off because the file was getting distracting. I'll turn it back on and run it.